occurrence-download/src/main/resources/templates/simple-sql-download/execute-simple-sql-query.ftl

```diff
@@ -0,0 +1,39 @@
+<#--
```

Comment:
this file does not need to be a template, can be moved to the hive-scripts dir, remove all ${r" and "}